sfmc_release_notes/SF_managed_clusters_ga_release_notes.md

```diff
@@ -0,0 +1,40 @@
+# Microsoft Azure Service Fabric managed clusters general availability release
+
+Azure Service Fabric managed clusters are now generally available. We first introduced Service Fabric managed clusters in October, 2020. Service Fabric managed clusters enable customers to deploy Service Fabric clusters as a single ARM resource, and reduce the complexity associated with common cluster operations such as scaling, and certificate rollovers. In this release we are bringing additional capabilities such as availability zone spanning, stateless node types, application secrets, application managed identities, and the ability to configure cluster upgrades.
+
+To utilize these new features, you must update the API version in the ARM template to *2021-05-01*.
+
+In this release, Service Fabric managed clusters are now available in all Azure public cloud regions and will roll out to Azure special clouds in the coming weeks.
+ 
+The following APIs are updated as part of this release:
+
+| API |
+|---------|
+| Microsoft.ServiceFabric/managedclusters |
+| Microsoft.ServiceFabric/managedclusters/nodetypes |
+| Microsoft.ServiceFabric/managedclusters/applications |
+| Microsoft.ServiceFabric/managedclusters/applicationTypes |
+| Microsoft.ServiceFabric/managedclusters/applications/services |
+
+## Features
+
+* [Availability zone spanning](https://docs.microsoft.com/azure/service-fabric/how-to-managed-cluster-availability-zones) - You can now deploy Service Fabric managed clusters across availability zones to increase resiliency against zone failures.
+* [Stateless node types](https://docs.microsoft.com/azure/service-fabric/how-to-managed-cluster-stateless-node-type) - Stateless node types enable faster node scaling, and VM patching when running services that do not require maintaining any state.
+* [Application secrets](https://docs.microsoft.com/azure/service-fabric/how-to-managed-cluster-configuration#enable-automatic-os-image-upgrades) - You can now access encrypted secrets from your Service Fabric application.
+* [Application managed identities](https://docs.microsoft.com/azure/service-fabric/how-to-managed-cluster-application-managed-identity.md) - Application managed identities enable you to access other Azure resources from your application.
+* [Cluster upgrade configurations](https://docs.microsoft.com//azure/service-fabric/service-fabric-cluster-upgrade) - You can now select between manual, and automatic upgrades and configure an upgrade window to meet the needs of your environment.
```

Comment:
typo in url //

You can now select between automatic(default) or manual upgrades and configure an upgrade window to meet the needs of your environment.